distributed/tests/test_scheduler.py

```diff
@@ -2201,3 +2201,16 @@ async def test_configurable_events_log_length(c, s, a, b):
     assert s.events["test"][0][1] == "dummy message 2"
     assert s.events["test"][1][1] == "dummy message 3"
     assert s.events["test"][2][1] == "dummy message 4"
+
+
+@gen_cluster()
+async def test_get_worker_monitor_info(s, a, b):
+    res = await s.get_worker_monitor_info()
+    ms = ("cpu", "time", "read_bytes", "write_bytes", "num_fds")
+    if WINDOWS:
+        ms = ("cpu", "time", "read_bytes", "write_bytes")
+    for w in (a, b):
+        for m in ms:
+            assert res[w.address]["range_query"][m] is not None
```

Comment:
What you have here is totally fine. I just wanted to point out that Python's built-in `all` function is sometimes useful to make a `for`-loop of `assert`s more compact

```python
assert all(res[w.address]["range_query"][m] is not None for m in ms)
```